src/runtime/virtcontainers/api.go

```diff
@@ -88,6 +88,14 @@ func createSandboxFromConfig(ctx context.Context, sandboxConfig SandboxConfig, f
 		return nil, err
 	}
 
+	// Now that we're in our 'final host cgroup' - let's make sure that the maximum number of
+	// CPUs reflects what is available to the shim.
+	currentMax := s.config.HypervisorConfig.DefaultMaxVCPUs
+	currentDefaultCPU := s.config.HypervisorConfig.NumVCPUs
```

Comment:
this'll be broken b/c the config will have applied a default already at LoadConfiguration time.  Need to revisit.  maybe there's not a benefit of checking max at all.